.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "workbench.colorTheme": "Solarized Dark",
+    "workbench.colorCustomizations": {
+        "activityBar.foreground": "#ff0000",
+        "activityBarBadge.foreground": "#000000",
+        "activityBarBadge.background": "#ff000095",
+        "sideBar.background": "#200000",
+    },
+    "python.pythonPath": "C:\\Users\\sean.micek\\AppData\\Local\\Continuum\\anaconda3\\envs\\cchecka\\python.exe"
+}
```

Comment:
Could you remove this file and squash the commit down?

Reply:
Ben, I had removed that file and added to gitignore before this latest PR https://github.com/ioos/compliance-checker/pull/870/files

Reply:
OK, I'm going to squash down these commits and merge.